src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,14 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = {
+                new Lottery().getRandomBall(),
+                new Lottery().getRandomBall(),
+                new Lottery().getRandomBall()
+        };
+
+        for (Ball ball: balls) {
+            System.out.println(ball.toString());
```

Comment:
Don't needlessly recreate lottery instances. Create one lottery instance before the for loop, then use that instance in the loop. We don't need to save the ball as a variable at all, just print the result of the .getRandomBall() method invocation